### DISCUSSION TITLE
[Feature] Passkey / FIDO2 support

### DISCUSSION THREAD

**Discussion by flosoft** [immich-app/immich#13513] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

It would be great to allow users to use Passkeys / FIDO2 instead of a password.

Support on both Web and Mobile would be amazing.

### Platform

- [X] Server
- [X] Web
- [X] Mobile

**Comment by mmomjian** [comment]
Thanks for the request. We offer Oauth for more advanced authentication. We probably won’t be adding this to Immich.
